jekyll/_data/sidenav.yml

```
@@ -1116,14 +1119,18 @@ ja:
         - name: Metrics and Monitoring
           i18n_name: メトリクスとモニタリング
           link: ja/2.0/server-3-operator-metrics-and-monitoring
+        - name: Introduction to Nomad
+          i18n_name: Nomad 概要
+          link: ja/2.0/server-3-operator-nomad
         - name: Configuring a Proxy
           i18n_name: プロキシの設定
           link: ja/2.0/server-3-operator-proxy
         - name: User Accounts
           i18n_name: ユーザー管理
           link: ja/2.0/server-3-operator-user-accounts
         - name: Orbs
-          i18n_name: Orbs
+          i18n_name: Managing Orbs
+          i18n_name: Orb　の管理
```

For consistency, i imagine this should again be `Orbs` not `Orb`.

```suggestion
          i18n_name: Orbs　の管理
```